tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     lint,
-    py{35,36,37,38},
+    py3{5,6,7,8},
```

Comment:
I am against, what if python4 is released?

Reply:
In which case, I'd be tempted to nest the braces: py{3{7,8,9},40} , but okay, I shall try and stop myself from injecting shell expansion into our tox config file. :-)